ql/src/test/results/clientpositive/llap/vector_const.q.out

```diff
@@ -40,7 +40,7 @@ STAGE PLANS:
                   alias: varchar_const_1
                   Statistics: Num rows: 1 Data size: 4 Basic stats: COMPLETE Column stats: COMPLETE
                   Select Operator
-                    expressions: 'FF' (type: varchar(3))
+                    expressions: 'FF' (type: varchar(4))
```

Comment:
interesting change; will this also mean that:
```
CONCAT(CAST('F' AS CHAR(200)),CAST('F' AS CHAR(200)))
```
will not be processable because it would need `CHAR(400)` - which is not supported?

Reply:
I have just checked and added a test. This gets converted into a STRING: It is the same behavior before/after the patch. If the precision of any single one of them is above 255, then an error message is thrown.